pkg/provider/github/github.go

```diff
@@ -253,10 +254,18 @@ func (v *Provider) SetClient(ctx context.Context, run *params.Run, event *info.E
 }
 
 // GetTektonDir Get all yaml files in tekton directory return as a single concated file
-func (v *Provider) GetTektonDir(ctx context.Context, runevent *info.Event, path string) (string, error) {
+func (v *Provider) GetTektonDir(ctx context.Context, runevent *info.Event, path, provenance string) (string, error) {
 	tektonDirSha := ""
 
-	rootobjects, _, err := v.Client.Git.GetTree(ctx, runevent.Organization, runevent.Repository, runevent.SHA, false)
+	v.provenance = provenance
+	// default set provenance from the SHA
+	revision := runevent.SHA
+	if provenance == "default_branch" {
+		revision = runevent.DefaultBranch
+		v.Logger.Infof("Using PipelineRun definition from default_branch: %s", runevent.DefaultBranch)
+	}
```

Comment:
How about adding info related to `Using PipelineRun definition from source branch` after the if condition to make sure from which branch pipeline run is triggered

Reply:
I think since it's implicit and is the default we don't need to add that, we are going to end up spamming the logs more than we do currently?

Reply:
I agree but i am just thinking from end user perspective like once they set the setting to default branch they see the info log and whet they remove setting there won't be any log to view 

I mean its not compulsory but as we introduced new functionality thought that adding that info will be more clear

Reply:
okay sounds good, i added it...

Reply:
in this forced change https://github.com/openshift-pipelines/pipelines-as-code/compare/6a36492d8d6dbf7ebac9219e3366b5b2d8e3e0b6..7d6dfa78a35320fded6c4efaaab5e7dde1f0c200